scene/resources/mesh.cpp

```diff
@@ -66,11 +66,11 @@ int Mesh::surface_get_face_count(int p_idx) const {
 	return 0;
 }
 
-int Mesh::get_face_count() const {
+int Mesh::get_triangle_count() const {
 	int faces_size = 0;
```

Comment:
```suggestion
	int triangle_count = 0;
```

Reply:
Ah yes, will fix that. Already done it in my working branch.